### PR TITLE
Remove double quotes from external table name

### PR DESCRIPTION
**Pull request by ravjotbrar**
### Summary

Prior to this change external tables were created under the public schema even if dbschema was specified. The bug was caused by double quoting the table name, which prepended the dbschema to the table name. 

### Description

Removed double quotes from create external table statement's table name. 

### Related Issue

https://github.com/vertica/spark-connector/issues/258

### Additional Reviewers

@alexr-bq 
@jonathanl-bq

### CODE REVIEW

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala [hunk 1]
```diff
@@ -250,12 +250,13 @@ class VerticaDistributedFilesystemWritePipe(val config: DistributedFilesystemWri
 
           if(inferStatement.contains(EscapeUtils.sqlEscape(s"${config.fileStoreConfig.externalTableAddress.stripSuffix("/")}/**/*.parquet")))  {
             logger.info("Inferring partial schema from dataframe")
-            schemaTools.inferExternalTableSchema(createExternalTableStatement, config.schema)
+            schemaTools.inferExternalTableSchema(createExternalTableStatement, config.schema, tableName)
           }
           else {
             logger.info("Inferring schema from parquet data")
```
Comment (alexey-temnikov) [inline connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala:256]: Shall we downgrade it to debug level?

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala [hunk 2]
```diff
@@ -250,12 +250,13 @@ class VerticaDistributedFilesystemWritePipe(val config: DistributedFilesystemWri
...
-            logger.info("The create external table statement is: " + createExternalTableStatement)
-            Right(createExternalTableStatement)
+            val updatedStatement = createExternalTableStatement.replace("\"" + tableName + "\"", tableName)
+            logger.info("The create external table statement is: " + updatedStatement)
```
Comment (alexey-temnikov) [inline connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemWritePipe.scala:258]: Shall it be printed in debug mode only?

File: examples/external-table-example/README.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
-# External Table Example
```
Comment (alexey-temnikov) [inline examples/external-table-example/README.md:1]: Why is it deleted?

File: functional-tests/src/main/scala/com/vertica/spark/functests/EndToEndTests.scala [hunk 4]
```diff
@@ -3595,7 +3595,7 @@ class EndToEndTests(readOpts: Map[String, String], writeOpts: Map[String, String
     fsLayer.removeDir(fsConfig.address)
     fsLayer.createDir(fsConfig.address, "777")
   }
-
+
```
Comment (alexey-temnikov) [inline functional-tests/src/main/scala/com/vertica/spark/functests/EndToEndTests.scala:3598]: nit: extra whitespace?

File: connector/src/main/scala/com/vertica/spark/util/schema/SchemaTools.scala [hunk 5]
```diff
@@ -436,11 +436,11 @@ class SchemaTools extends SchemaToolsInterface {
...
   }
 
-  def inferExternalTableSchema(createExternalTableStmt: String, schema: StructType): ConnectorResult[String] = {
-    println("The passed in create external table statement is: \n" + createExternalTableStmt)
-    val indexOfOpeningParantheses = createExternalTableStmt.indexOf("(")
-    val indexOfClosingParantheses = createExternalTableStmt.indexOf(")")
-    val schemaString = createExternalTableStmt.substring(indexOfOpeningParantheses + 1, indexOfClosingParantheses)
+  def inferExternalTableSchema(createExternalTableStmt: String, schema: StructType, tableName: String): ConnectorResult[String] = {
+    val stmt = createExternalTableStmt.replace("\"" + tableName + "\"", tableName)
+    val indexOfOpeningParantheses = stmt.indexOf("(")
```
[the remark this answers is not in the document]
Comment (ravjotbrar) [reply connector/src/main/scala/com/vertica/spark/util/schema/SchemaTools.scala:441]: There won'y be any nested statements returned with vertica's infer schema function, as it returns a [create external table statement](https://www.vertica.com/docs/9.3.x/HTML/Content/Authoring/SQLReferenceManual/Statements/CREATEEXTERNALTABLEASCOPY.htm)

**Pull request merged**